chart/templates/deployment.yaml

```diff
@@ -0,0 +1,121 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: poc-acapy-multitenant
+  labels:
+    app: poc-acapy-multitenant
+    chart: poc-acapy-multitenant-0.1.0
+    release: poc-acapy-multitenant
+    heritage: Helm
+spec:
+  replicas: 1
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+  selector:
+    matchLabels:
+      app: poc-acapy-multitenant
+      release: poc-acapy-multitenant
+  template:
+    metadata:
+      annotations:
+        # needs to be updated I suppose
+        iam.amazonaws.com/role: arn:aws:iam::715299534867:role/gid-dev-poc-acapy-multitenant
```

Comment:
Do you need to access any of the AWS resources or why do you have a IAM role provided ?

Reply:
I don't think so, so can just remove it